pages/linux/strace.md

```diff
@@ -0,0 +1,16 @@
+# strace 
+
+> Troubleshooting tool for tracing system calls
+
+- Start tracing a specific PID 
+
+`strace -p {{pid}}`
+
+- Trace a process and filter output by system call
+
+`strace -e {{system call name}} -p {{pid}}`
+
+- Start tracing and produce a summary of system calls at the end, instead of showing individual calls.
```

Comment:
I'd use the term "cumulative" somewhere in the description (maybe "cumulative summary"), to help with memorizing the option name.